libcontainer/specconv/spec_linux.go

```diff
@@ -494,6 +494,17 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 				Ambient:     spec.Process.Capabilities.Ambient,
 			}
 		}
+		if spec.Process.Scheduler != nil {
+			config.Scheduler = &configs.Scheduler{
```

Comment:
As the description in https://github.com/opencontainers/runc/pull/3962#discussion_r1291710834
```
Since spec.Process.Scheduler is of the same type as config.Scheduler, you can just assign it directly, no need to go field-by-field.
```
@kolyshkin 
But I think we should not assign it directly, because if someone modify these value in `spec.Process.Scheduler` in other place by a mistake, the value in `config.Scheduler` will be changed too.
So, use an new strcture will be more safe.

Reply:
In case you worry that somehow the original structure will be modified, you can use a copy via deferefence, like this:

```go
if spec.Process.Scheduler != nil {
	s := *spec.Process.Scheduler // copy
	config.Scheduler = &s
}
```

which is not perfect but at least we don't have to explicitly name every field.